plugins/run/help-functions

```diff
@@ -28,7 +28,7 @@ fn-help-content() {
   declare desc="return help content"
   cat <<help_content
     run [-e|--env KEY=VALUE] [--no-tty] <app> <cmd>, Run a command in a new container using the current app image
```

Comment:
The flag also is here right?

Reply:
It will work as-is (i.e. the flag will be parsed), but seems redundant and maybe it doesn't make sense to add it to the description?
You either have tty already (and it will be used) or you are executing something from a non-terminal environment. In the second case if you do need tty you will need to attach it externally, in which case you might as well go with run:detached.